helm/prometheus-meta-operator/templates/prometheus-rules/deployment.management-cluster.rules.yml

```diff
@@ -82,11 +82,37 @@ spec:
       annotations:
         description: '{{`Deployment {{ $labels.namespace}}/{{ $labels.deployment }} is not satisfied.`}}'
         opsrecipe: deployment-not-satisfied-china/
-      expr: kube_deployment_status_replicas_unavailable{cluster_type="management_cluster", deployment!~"coredns|nginx-ingress-controller|.+-admission-controller-unique", cluster_id=~"argali|giraffe"} > 0
+      expr: kube_deployment_status_replicas_unavailable{cluster_type="management_cluster", deployment!~"coredns|nginx-ingress-controller|alertmanager.*|prometheus.*|promxy.*|.+-admission-controller-unique", cluster_id=~"argali|giraffe"} > 0
```

Comment:
Sneaky 😉 OK, I'm going to pretend I didn't see that 🙈

Reply:
Not really sneaky as this was updated between my first and last rule generation :D